app/policies/comment_policy.rb

```diff
@@ -1,5 +1,5 @@
 class CommentPolicy < ApplicationPolicy
-  authorize :user, allow_nil: true
+  authorize :user
```

Comment:
It is redundant. https://actionpolicy.evilmartians.io/#/custom_policy?id=custom-base-policy
```
  # ActionPolicy::Policy::Defaults module adds the following

  authorize :user
```